services/kommander/0.2.0/defaults/cm.yaml

```diff
@@ -56,12 +56,12 @@ data:
           manager:
             rootCertSecretName: ${caSecretName}
             image:
-              tag: ${licensingControllerManagerImageTag}
-              repository: ${licensingControllerManagerImageRepository}
+              tag: ${kommanderLicensingControllerManagerImageTag:=v2.2.0-dev}
```

Comment:
This is fixed in https://github.com/mesosphere/kommander/pull/1512.

Reply:
I double-checked the PR you linked and I do not understand how it solves the issue I described in PR descrption:

```
we need to define a default version for kommander-licensing subchart as it does not define sane defaults by itself. Cleaning it up is beyond the scope of the UniversalUpdater work.
```
The code in question is here:

https://github.com/mesosphere/kommander/blob/aa264652835ce100995e53cf7e338ffae59e6e1f/chart/kommander/charts/kommander-licensing/values.yaml#L17
https://github.com/mesosphere/kommander/blob/aa264652835ce100995e53cf7e338ffae59e6e1f/chart/kommander/charts/kommander-licensing/values.yaml#L46

and the changes in the PR you pointed are basically doing:
```diff
-appVersion: "dev"
+appVersion: v0.0.0-dev
description: Kommander Licensing
-version: 0.2.0-dev
+version: v0.0.0-dev
```

Also, please note that we need to make the image tags overridable, hence we can't simply change appVersion to `v2.2.0-dev`. We probably could clean this up in a better way but we are already pushing the scope of the `UniversalMigrator` change and this work is blocking other work. Let's please merge what we have and iterate/prioritize remaining work.

Reply:
The chart packaging changes the version of the commander-licensing subchart when it is packaged (see https://github.com/mesosphere/kommander/pull/1512/commits/33ab8ae6dc9999c3374368c73b63ce235b4d3aa1). This means that the defaulting of the image tag in kommander-licensing works properly.

Reply:
So the image tags in the kommander-licensing chart use the chart version, which now matches the parent kommander chart version by virtue of how we package the chart. You can still override the tags by providing relevant values, but generally this is not necessary, the same as it is not necessary for the kommander chart.

Reply:
I've removed this now.